arangod/IResearch/IResearchViewCoordinator.cpp

```diff
@@ -397,6 +440,33 @@ bool IResearchViewCoordinator::emplace(
 
       return nullptr;
     }
+
+    // create links - "on a best-effort basis"
+    if (info.hasKey("links")) {
```

Comment:
this should really be in a try-catch since the view definition has already been persisted

Reply:
Please elaborate on that.

Reply:
as far as the storage engine is concerned the view exists, so upon recovery the view will be instantiated
if however there is an exception during link creation then the current arangod process will report that the view has not been created, which would be inconsistent with recovery